lab_5_scraper/scraper.py

```diff
@@ -101,6 +200,15 @@ def make_request(url: str, config: Config) -> requests.models.Response:
     Returns:
         requests.models.Response: A response from a request
     """
+    sleep(15)
```

Comment:
it should be random and can be much smaller, otherwise it would take so much time